tox.ini

```diff
@@ -48,7 +48,7 @@ commands =
 
 [testenv:docs-lint]
 description = run linter (rstcheck) to ensure there aren't errors on our docs
-deps = -r{toxinidir}/requirements/docs-lint.txt
+deps = -r{toxinidir}/requirements/docs.txt
```

Comment:
Don't we still need rstcheck?

Reply:
Looks like it's already in `docs.txt` -- so we should remove the docs-lint file.

Reply:
(removed in a commit)